src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,19 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        List<Integer> evenNumbers = numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
+                .map(Integer::parseInt)
+                .filter(number -> number % 2 == 0)
+                .toList();
+
+        if (evenNumbers.isEmpty()) {
+            throw new RuntimeException("Can't get min value from list" + numbers);
+        }
+
+        return evenNumbers.stream()
+                .min(Comparator.naturalOrder())
+                .orElseThrow(NoSuchElementException::new);
```

Comment:
we need to have IntStream to be able to use it's methods without a comparator and not split the stream into 2 parts 
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(number -> number % 2 == 0)
                .min()
                .orElseThrow(() -> new RuntimeException(("Can't get min value from list" + numbers));
```